### ISSUE TITLE
Create a feature that randomly chooses 5 of 10 "suggested apps" to display on /Use

### ISSUE THREAD

**Issue by jjmstark** [ethereum/ethereum-org-website#63]
We would like to have a feature where:

- We have a pre-determined list of 10 applications built on ethereum
- Every time a user loads the page, 5 of those applications are chosen at random
- Those 5 are displayed in a random order on https://ethereum.org/use/, under "Use an application built on Ethereum"

**Comment by maurelian** [comment]
Looking at #23, I can see how this will be a regular item for consideration. Is there someway of deferring the curation to another source? There's a lot of dapp listing site out there, maybe one has an API we can pull from, or we can just link out to them. 

If we go the direction of "just include everything that's not an obvious scam", I'd be in favor of modifying this feature to: 

* We have a huge list of applications built on ethereum
* Every time a user loads the page, 5 of those applications are chosen at random
* Those 5 are displayed in a random order on https://ethereum.org/use/, under "Use an application built on Ethereum"
* There's a link to "view more apps", which has the full list.

**Comment by jjmstark** [comment]
I propose we keep the list small and tightly curated for the immediate future, for the following reasons:

- Committing to "everything that's not a scam" still creates a very large surface area for review (there are ~1,900 listed on stateofthedapps.com). 
- If we ever make a mistake and do list a scam, we've created a horrible experience for some person who used ethereum.org. 
- The purpose of linking to these apps on /Use is to actually show the user something interesting they can "do with Ethereum". If that's the goal, a curated list of high-quality applications - with good landing pages, good UX, known to be non-scams - seems to achieve it better. 
- We already achieve part of the "be comprehensive" goal by linking to other people's lists further down this page.

**Comment by jjmstark** [comment]
One valuable thing to do here would be to define the criteria we're using to put applications on this list. A few starting points:

- Is the app a "user" application? There are many applications or services "built on ethereum" that are narrowly targeted at developers or sophisticated users. For instance, we would not list Infura here (though it is, and should be, on the /Build page)
- Does the app have a good user on-boarding process? Can a user follow our link, and find all the instructions they need to get started on the applications website?
- Does the application "round out" the list by adding a new *kind* of application not already present?

**Comment by maurelian** [comment]
Both of the comments are great. 
I think the only criterion I would add is:

* Is it a popular / well-established app?

That's subjective, but cuts out the incentive for a long tail of apps trying to get a free listing.

**Comment by maurelian** [comment]
The only criterion I'd add is: 

* Should be a popular and established application. 

Yes, it's insider-ish, but it saves a lot of effort in reviewing and fending off listings from unknown/unproven apps.

**Comment by texture** [comment]
If this feature exists, it should be used to get project sites to put some sort of "built on ethereum" reference onto their site for inclusion.

**Comment by samajammin** [comment]
Good discussion here. Looks like it's clarified our acceptance criteria in the README:
https://github.com/ethereum/ethereum-org-website#notes-on-individual-sub-pages

It's on our roadmap to build a generic component that handles this random selection (i.e. given a list of X URLs & anchor text, display a random list of Y links). There are a couple implementation details to sort through - e.g. should the list randomize every time the component renders? or only once per user session? Feel free to add thoughts in this discussion.

It's not a top priority but we'll update progress here (i.e. label as `wip` when work begins). Until then, labeling this as a good issue for contributors to pick up!

**Comment by samajammin** [comment]
Closing in favor of https://github.com/ethereum/ethereum-org-website/issues/352
